os/board/imxrt1020-evk/src/imxrt_bringup.c

```diff
@@ -269,16 +271,16 @@ int imxrt_bringup(void)
 	}
 #endif
 
-#if defined(CONFIG_I2C_DRIVER) && defined(CONFIG_IMXRT_LPI2C1)
+#if defined(CONFIG_I2C) && defined(CONFIG_IMXRT_LPI2C1)
```

Comment:
I believe You already know how to reduce executing code of below case, right?
```
if (A & B)
{
   do_somethingB;
}

if (A && C)
{
  do_somethingC;
}
....
if (A && F)
{
  do_somethingF;
}
```
System will check validation of A for everytime. but you can improve it =)

Reply:
Also i2c is kind of bus(lower), it should be initialized when upper is needed, not bring up time.